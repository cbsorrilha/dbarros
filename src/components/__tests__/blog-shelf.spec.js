import React from "react"
import renderer from "react-test-renderer"
import BlogShelf from '../blog-shelf';


describe('Snapshot testing the BlogShelf', () => {
    it("renders correctly", () => {
        const props = {}
        const tree = renderer.create(<BlogShelf {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
