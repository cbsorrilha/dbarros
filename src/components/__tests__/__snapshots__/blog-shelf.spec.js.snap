// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the BlogShelf renders correctly 1`] = `
<div
  className="sc-AykKC edhYBg"
/>
`;
